Mohammed/index.js

```diff
@@ -0,0 +1,42 @@
+
+// 1. NaN
+
+// a-
+// NaN stands for 'Not A Number'
+
+// b-
+let checkForNaN = 'hello' * 1;
+console.log((checkForNaN !== NaN) ? 
+`The expression ("hello" * 1) is a ${checkForNaN}` : `The expression ("hello" * 1) is a ${checkForNaN}`);
+
+// c-
+checkForNaN = '100em' *1;
```

Comment:
Here also the same. We wanna check if the result is true or false! You also dont need *1 in this task!
console.log(isNaN("100em"));

Reply:
Hi Ciamac, thank you for your feedback. I forgot about the isNaN() method so I went to reinvent it!
I pushed the refined code to github.